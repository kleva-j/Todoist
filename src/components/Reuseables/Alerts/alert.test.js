import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { CustomAlert } from '.';

configure({ adapter: new Adapter() });

describe('CustomAlert', () => {
  const testProps = {
    type: 'danger',
    message: 'This is a message',
    description: 'This is a description',
    afterClose: jest.fn(),
  };

  const wrapper = shallow(<CustomAlert {...testProps} />);
  const { type, message, description } = wrapper.props();

  test('should render correctly', () => {
    expect(type).toBe(testProps.type);
    expect(message).toBe(testProps.message);
    expect(description).toBe(testProps.description);
  });
  
  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
