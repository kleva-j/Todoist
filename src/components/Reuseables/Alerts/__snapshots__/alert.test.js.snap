// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomAlert should match snapshot 1`] = `ShallowWrapper {}`;
